src/app/global/not-found/not-found.component.scss

```diff
@@ -0,0 +1,15 @@
+.not-found {
+  min-height: 100vh;
```

Comment:
min-height: calc(100vh - var(--header-height));

this makes it equal to the height of the screen